Content.Server/Paper/PaperSystem.cs

```diff
@@ -188,6 +214,83 @@ public bool TryStamp(EntityUid uid, StampDisplayInfo stampInfo, string spriteSta
             return true;
         }
 
+        // BEGIN OF UMBRA ADDITIONS
+        // Umbra: Send paper signing alt verb to the client if applicable.
+        // Based on LockSystem.cs for alt-click behavior.
+        private void AddSignVerb(EntityUid uid, PaperComponent component, GetVerbsEvent<AlternativeVerb> args)
```

Comment:
What would the T be here? I just followed `LockSystem.cs` for this function signature.

Reply:
(This is not a big deal at all, I just wanted to point it out in case you did not know) 

The T is the component, so in this case it would be `Entity<PaperComponent>`.